docs/SettingUp.md

```diff
@@ -0,0 +1,37 @@
+# Control Your Crowd - Setting up
+## Prerequisites
+1. JDK `11` or above
```

Comment:
Do you think its a good idea to provide the link to download the Java 11?
https://www.oracle.com/sg/java/technologies/javase-jdk11-downloads.html